spec/unit/network/http/handler_spec.rb

```diff
@@ -95,9 +95,6 @@ def respond(text)
       expect(res[:content_type_header]).to eq("application/json")
       expect(res_body["issue_kind"]).to eq(Puppet::Network::HTTP::Issues::RUNTIME_ERROR.to_s)
       expect(res_body["message"]).to eq("Server Error: the sky is falling!")
-      expect(res_body["stacktrace"].is_a?(Array) && !res_body["stacktrace"].empty?).to be_truthy
```

Comment:
I assumed that the json schema validation would fail, but I didn't actually try that. Lemme check that.